examples/svelte-example/src/App.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+    import { onDestroy, onMount } from 'svelte';
+    import { QueryStatus } from '../../../dist';
+    import { counterApi } from './services/counter';
+    import { store } from './store';
+
+    const { incrementCount, decrementCount } = counterApi.mutationActions;
+
+    let data;
+    let status;
+    let error;
+
+    const selector = counterApi.selectors.query.getCount();
+    const unsubscribeSelector = store.subscribe(() => ({ data, status, error } = selector(store.getState())));
+
+    const increment = async () => ({ payload: data } = await store.dispatch(incrementCount(1)));
+
+    const decrement = async () => ({ payload: data } = await store.dispatch(decrementCount(1)));
```

Comment:
The invalidates wasn't causing a refetch here, so i'm just reassigning it with the response. Note: the payload didn't have the type come through even though it knows about `CountResponse`. In general, I have no idea if this is the right thing to do 🙃

Reply:
Okay - I realized I'm a big ol' dumbdumb and _literally_ no-op'd the code that runs the updates 🤦